source/includes/reference/_app_webhooks.md

```diff
@@ -0,0 +1,209 @@
+# Application Webhooks
+
+Application webhook subscription is a mechanism that will allow you to receive notifications to your servers whenever various events happen in relation to different resources created by an ***application***. You can subscribe to receive webhooks about resource created under a specific [***profile***](#profile-webhooks).
+
+Before proceeding, make sure the endpoint where you intend to receive webhooks satisfies the following requirements:
+
+* Has a valid domain name (IPs are disallowed)
+* Listens to HTTPS requests on port `443`
+* Has a valid HTTPS certificate signed by a trusted Certificate Authority - CA (self-signed or expired certificates are not accepted)
+* Does not include any query arguments in the URL
+
+`https://webhooks.example.com/balance-change` is a valid URL; `http://webhooks.example.com:8080/hook.php?type=balance` is not.
+
+You can have multiple subscriptions per event type though be mindful you will receive duplicate callbacks, one for each subscription.
+
+> **NB! Please note that you have to use client level token for making following requests.**
```

Comment:
Pedantic mode: "NB" basically means "please note"

Reply:
"client level tokens"

Reply:
is it worth explaining what a client level token is? in the docs it's only mentioned in one place: https://api-docs.transferwise.com/#users-sign-up-with-registration-code